scene/gui/rich_text_label.cpp

```diff
@@ -36,6 +36,33 @@
 #include "editor/editor_node.h"
 #endif
 
+static bool _is_symbol(CharType c) {
+	return c != '_' && ((c >= '!' && c <= '/') || (c >= ':' && c <= '@') || (c >= '[' && c <= '`') || (c >= '{' && c <= '~') || c == '\t' || c == ' ');
```

Comment:
I'd put this alongside similar functions in ustring.cpp just to keep similar stuff together.
https://github.com/godotengine/godot/blob/5c636875e418fbf055ddcff8d682639d1f096d05/core/ustring.cpp#L54-L57

Reply:
Moved `is_symbol` to `ustring`.

Also deduplicated `select_word` and moved it to `ustring` as well.